src/components/ProductPage/ProductWalletBalance.tsx

```diff
@@ -16,14 +19,29 @@ const ProductWalletBalance: React.FC<ProductWalletBalanceProps> = ({
   latestPrice = 0,
   currentBalance = 0,
 }) => {
+  const wallet = useWallet()
+  const handleAddToMetamask = useAddToMetamask()
+
   return (
     <ProductPageSection title='My Assets'>
-      <StyledTokenValuation>
-        ${numeral(latestPrice * Number(currentBalance)).format('0.00a')}
-      </StyledTokenValuation>
-      <StyledTokenBalance>
-        {numeral(currentBalance).format('0.000a')} {symbol}
-      </StyledTokenBalance>
+      <div>
+        <StyledTokenValuation>
+          ${numeral(latestPrice * Number(currentBalance)).format('0.00a')}
+        </StyledTokenValuation>
+        <StyledTokenBalance>
+          {numeral(currentBalance).format('0.000a')} {symbol}
+        </StyledTokenBalance>
+        <AddToMetamask>
+          <Button
+            full
+            size={'sm'}
+            text='Add to Metamask'
+            variant={'default'}
+            disabled={wallet.ethereum == undefined}
+            onClick={() => handleAddToMetamask(symbol)}
+          />
+        </AddToMetamask>
```

Comment:
IMO the add to metamask button while logged out is a bit too prominent. You can't really tell it's disabled. Maybe a gray background or something? Or remove it when logged out?
![image](https://user-images.githubusercontent.com/7955024/113335930-806bb400-92da-11eb-8c71-a6663314a0d7.png)